packages/contracts/test/Verifier.ts

```diff
@@ -2,93 +2,156 @@ import { loadFixture } from '@nomicfoundation/hardhat-toolbox/network-helpers'
 import { expect } from 'chai'
 import { ethers } from 'hardhat'
 import {
-  splitToWords,
-  exportCallDataGroth16,
-  extractWitness,
+  InitArgs,
+  init,
+  generateArgs,
+  prove,
   WASM_URL,
   ZKEY_URL,
+  VK_URL,
+  AnonAadhaarProof,
+  PackedGroth16Proof,
+  packGroth16Proof,
 } from '../../core/src'
-import crypto from 'crypto'
-import { fetchKey } from './util'
-import { genData } from '../../core/test/utils'
+import { testQRData } from '../../circuits/assets/dataInput.json'
 import fs from 'fs'
 
 describe('VerifyProof', function () {
   this.timeout(0)
 
+  let packedGroth16Proof: PackedGroth16Proof
+  let anonAadhaarProof: AnonAadhaarProof
+  let certificate: string
+  let user1addres: string
+
+  this.beforeAll(async () => {
+    const certificateDirName = __dirname + '/../../circuits/assets'
+    certificate = fs
+      .readFileSync(certificateDirName + '/uidai_prod_cdup.cer')
```

Comment:
I think we should export gov certification.  Because this public data and developer also use it for check something(like check signature before gen the proof). 
I remember we have 2 or 3 certs.

Reply:
I didn't fully understand, do you mean allowing importing certificates from the SDK?

Reply:
Yes, so developer don't need find it by themself.

Reply:
I'm not sure it's the proper way to handle it there. Also, it might not be in the scope of this PR.

Reply:
Got it! I think this useful because developer need cert to gen the proof. 
Lets do it in another PR!